src/isIsogram.test.js

```diff
@@ -6,4 +6,34 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`the empty string should be an isogram`, () => {
```

Comment:
Change the test name because in the summary you should describe the function, e.g. should return `true` for an empty string